helper.php

```diff
@@ -266,7 +269,11 @@ function _get_instructions($page, $sect, $mode, $lvl, $flags, $root_id = null, $
                 global $ID;
                 $backupID = $ID;
                 $ID = $page; // Change the global $ID as otherwise plugins like the discussion plugin will save data for the wrong page
-                $ins = p_cached_instructions(wikiFN($page), false, $page);
+                if ($flags['noinstructionscache']) {
```

Comment:
This will trigger a notice if the flag is not set, please add an initialization to `$this->defaults` in the constructor.